composer.json

```diff
@@ -0,0 +1,135 @@
+{
+    "name": "drupal/recommended-project",
```

Comment:
We should change this. And maybe have the `template-prune` task set it (using `composer config name`).